Makefile

```diff
@@ -16,7 +16,7 @@ BPF_FILES_EVAL := $(shell git ls-files $(ROOT_DIR)/bpf/ | grep -v .gitignore | t
 BPF_FILES ?= $(BPF_FILES_EVAL)
 BPF_SRCFILES := $(subst ../,,$(BPF_FILES))
 
-SWAGGER_VERSION := v0.19.0
+SWAGGER_VERSION := v0.20.1
```

Comment:
Only this one @nebril , the rest was all auto generated. I mean there were some necessary refactoring in our code because swagger changed the name of `IPAM` to `Ipam`.